solutions/LeBron.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+grep -o 'LeBron.*' puzzle/LeBronInfo.txt \
+  | sed -n '50p' \
+  | sed 's/^LeBron\s*//'
```

Comment:
This can be simplified to a single `sed` script:

```suggestion
sed -n '50s/^LeBron: //p' puzzle/LeBronInfo.txt
```

I.e. turn off automatic printing and then execute the replace and print command only on line 50.